extensions/amp-analytics/0.1/activity-impl.js

```diff
@@ -318,18 +318,22 @@ export class Activity {
     return this.activityHistory_.getTotalEngagedTime(secondsSinceStart);
   }
   /**
-   * Get the incremental engaged time since the last push and reset it.
+   * Get the incremental engaged time since the last push and reset it if asked.
    * @param {string} name
+   * @param {string} reset
    * @return {number}
    */
-  getIncrementalEngagedTime(name = '') {
+  getIncrementalEngagedTime(name = '', reset = 'true') {
```

Comment:
I wonder if it is easier to just create another increamentalEnagedTime value with a different name and not adding the reset value. 
But I am good with adding `reset`. Please add test and documentation.

Reply:
I started off creating a separate function but I ended up doing this instead. My thinking was that since I would have to expose it in so many places adding the flag as a function argument was a cleaner implementation.